cli/parse-options.js

```diff
@@ -34,12 +38,26 @@ function parseOptions (options) {
     config.url = options.url
   }
 
+  PouchDB.plugin(require('pouchdb-mapreduce'))
+
   if (options.dbUrl) {
-    config.db.url = options.dbUrl
-  }
-  if (options.inMemory) {
+    if (!urlParse(options.db.url).auth) {
+      return next(new Error('Authentication details missing from database URL: ' + options.db.url))
```

Comment:
@gr2m I changed it! And I want to ask about test coverage. Can I try to make a code to cover lines in `cli/parse-option.js`?